db/seeds/articles.rb

```diff
@@ -77,7 +77,8 @@
       image:          image,
       status_id:      published_status.id,
       content_format: "html",
-      hide_layout:    true
+      hide_layout:    true,
+      short_path: (0...8).map { ('a'..'z').to_a[rand(26)] }.join
```

Comment:
Since it doesn't actually need to be "short", it's safer to use `SecureRandom.hex`.